nyxx/lib/src/events/ThreadMembersUpdateEvent.dart

```diff
@@ -0,0 +1,21 @@
+part of nyxx;
+
+/// Fired when a thread has a member added/removed
+class ThreadMembersUpdateEvent {
+  late final _ChannelCacheable<ThreadChannel> thread;

```

Comment:
Better replace that with CacheableTextChannel which implements cacheable and TextChannel interface

Reply:
Not found